src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/IMvcBuilder.cs

```diff
@@ -12,5 +14,10 @@ public interface IMvcBuilder
         /// Gets the <see cref="IServiceCollection"/> where MVC services are configured.
         /// </summary>
         IServiceCollection Services { get; }
+
+        /// <summary>
+        /// Gets a list of <see cref="ApplicationPart"/> instances that compose the application.
```

Comment:
This comment isn't quite accurate. The manager does more than just have a list of parts.